.changeset/polite-taxis-repair.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vercel/edge-config': minor
+---
+
+Add the `consistentRead` option to allow reading from the origin
```

Comment:
Let's be clear this is not recommended even here in the changelog, as it will have way higher latency and can not handle the same read volume.